### PR TITLE
Grafana installed together with prometheus and Envoy Filter

### PR DESCRIPTION
**Pull request by matined**

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -30,15 +30,17 @@ jobs:
...
-      - name: Add Helm repos
+      - name: Deploy Prometheus And Grafana
         run: |
           helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-          helm repo add grafana https://grafana.github.io/helm-charts
           helm repo update
-          helm upgrade --install tyro-prometheus prometheus-community/prometheus --atomic --namespace tyro-monitoring
-          helm upgrade --install tyro-grafana grafana/grafana --atomic -f ./grafana/values.yaml
+          helm upgrade \
+            --install tyro-prometheus prometheus-community/kube-prometheus-stack \
```
Comment (HermanPlay) [inline .github/workflows/deploy.yaml:38]: Why we install kube-prometheus-stack instead of Prometheus?

File: chart/templates/istio/envoy.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+apiVersion: networking.istio.io/v1alpha3
```
Comment (HermanPlay) [inline chart/templates/istio/envoy.yaml:1]: What this config makes?

**Review by HermanPlay — COMMENTED** [review]
Please answer my questions 🥺

File: chart/templates/istio/envoy.yaml (hunk 2 above)
Comment (matined) [reply chart/templates/istio/envoy.yaml:1]: It allows to add Envoy Filters, which can run WASM scripts. This one adds a filter thats runs AFTER response and should log the response.

It doesnt work yet, but I want to do it incrementally

File: .github/workflows/install_istio.yaml [hunk 3]
```diff
@@ -31,7 +31,7 @@ jobs:
         curl -L https://istio.io/downloadIstio | sh -
         cd istio-*
         export PATH=$PWD/bin:$PATH
-        istioctl install -y
+        istioctl install -y --set meshConfig.accessLogFile=/dev/stdout
```
[the remark this answers is not in the document]
Comment (matined) [reply .github/workflows/install_istio.yaml:34]: standard console output, the one that you see using `kubectl logs ...`

**Review by HermanPlay — APPROVED** [review]
Great 👍

**Pull request merged**